ci/install.sh

```diff
@@ -11,13 +11,13 @@ main() {
         target=x86_64-apple-darwin
     fi
 
-    # TODO At some point you'll probably want to use a newer release of `cross`,
-    # simply change the argument to `--tag`.
+    local tag="$(git ls-remote --tags --refs --exit-code https://github.com/japaric/cross | cut -d/ -f3 | tail -n1)"
```

Comment:
With a `| grep '0\.1\.[0-9]\{1,\}$'` before the `tail` we can select the 0.1.x versions